api/src/changeRequests/processChangeRequest.spec.ts

```diff
@@ -118,4 +119,58 @@ describe("processChangeRequest", () => {
         expect(processResult.message).toBe("Document is identical to the one in the database");
         expect(processResult.changes).toBeUndefined();
     });
+
+    it("can validate a unique slug for a content document that does not exists", async () => {
+        const changeRequest = changeRequest_content();
+        changeRequest.doc.parentId = "post-post1";
+        changeRequest.doc._id = "test-slug-1";
+        changeRequest.doc.slug = "this-is-a-test-slug";
+
+        const res = await processChangeRequest("", changeRequest, ["group-super-admins"], db);
+        const dbDoc = await db.getDoc(changeRequest.doc._id);
+
+        expect(res.ok).toBe(true);
+        expect(dbDoc.docs[0].slug).toBe("this-is-a-test-slug");
+    });
+
+    it("can validate a unique slug for a content document that exists", async () => {
+        // This test relies on the previous test to have run first
```

Comment:
Not a huge fan of this as you never know if the order of tests is guaranteed - maybe Jest will change it in a future version. Instead I would suggest to specifically insert a different item into the database at the start of this test and then use that to verify it.